### PR TITLE
[RSS-32] Create Profile Page

### PR DESCRIPTION
**Pull request by comalvirdi**
Created Profile Page with limited styling
Screen Shot 2021-02-01 at 7 10 31 PM

### CODE REVIEW

File: src/components/ProfilePage/ProfilePage.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+export default (UserData) => (
+  <div>
+    <UserInfoContainer>
+      <StyledTitle>
+        {UserData.Name}
+        {UserInfo.Name}
+      </StyledTitle>
+      <StyledSubtitle2>{UserInfo.Email}</StyledSubtitle2>
+      <StyledSubtitle2>{UserInfo.CellPhoneNumber}</StyledSubtitle2>
+      <StyledButton variant="link">Edit contact information &gt; </StyledButton>
```
Comment (jackfales) [inline src/components/ProfilePage/ProfilePage.jsx]: Can you add an onClick handler that pushes the user to `/editprofile`? You need to use useHistory from 'react-router-dom'. Let me know if you have any questions or need any help.

**Review by jackfales — COMMENTED** [review]
Looks good, just push the user to /editprofile when they click on the corresponding link. Re-request review when the changes are made.

File: src/components/ProfilePage/ProfilePage.jsx (hunk 1 above)
Comment (comalvirdi) [reply src/components/ProfilePage/ProfilePage.jsx]: I did it without use history by using the link component from react-router-dom. It works, are you ok with this implementation?

**Review by jackfales — APPROVED** [review]
Nice work, I had no idea Bootstrap Links could redirect to routes so easily like that.

**Pull request merged**